lenovo/thinkpad/x260/default.nix

```diff
@@ -0,0 +1,20 @@
+{
+  imports = [
+    ../.
+    ../acpi_call.nix
+    ../../../common/cpu/intel
+  ];
+
+  # See https://linrunner.de/en/tlp/docs/tlp-faq.html#battery https://wiki.archlinux.org/index.php/TLP#Btrfs
+  services.tlp.extraConfig = ''
+    SATA_LINKPWR_ON_BAT=max_performance
```

Comment:
Good catch! We should probably apply this for all profiles, not just ThinkPad X260.